php_xdebug.h

```diff
@@ -104,7 +104,7 @@ struct xdebug_base_info {
 	xdebug_llist *filters_code_coverage;
 
 	struct {
-		zend_long     max_nesting_level;
+		zend_long max_nesting_level;
```

Comment:
Please don't make unrelated white space changes.

Reply:
I checked the history and these more whitespaces are leftover when some other vars were removed, but reverted